api/v1alpha1/condition_consts.go

```diff
@@ -37,3 +37,13 @@ const (
 	// DeletedReason (Severity=Info) documents a condition not in Status=True because the underlying object was deleted.
 	DeletedReason = "Deleted"
 )
+
+// Conditions related to the VirtualMachineImages
+const(
+	// VirtualMachineImageOsTypeNotSupportedCondition denotes that the OS type in the virtual machine image object is
```

Comment:
These should be positive values so don't use Not.

https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/proposals/20200506-conditions.md
> "Condition types MUST have a consistent polarity (i.e. "True = good");